common/djangoapps/student/models.py

```diff
@@ -179,6 +179,17 @@ class UserStanding(models.Model):
     standing_last_changed_at = models.DateTimeField(auto_now=True)
 
 
+class UserStandingConfig(ConfigurationModel):
+    disabled = models.TextField(
+        help_text="White-space separated list of usernames of users who should be disabled",
```

Comment:
How many usernames do we expect ended up in this list? I realize we probably have no idea. But it would only take 10s of usernames in this list before it became unmanageable to a non-coder. Is there a way to perhaps alphabetize it on an addition?